dlc_developer_config.toml

```diff
@@ -43,8 +43,8 @@ do_build = true
 [test]
```

Comment:
Please add `pytorch` to the `build_frameworks` variable. By default, we do not build any frameworks if this parameter is empty.
You can optionally set `build_inference = false` since this is a Training DLC PR